runtime/lua/vim/uri.lua

```diff
@@ -53,16 +53,16 @@ do
 end
 
 ---@private
-local function is_windows_file_uri(uri)
-  return uri:match('^file:/+[a-zA-Z]:') ~= nil
+local function in_windows()
+  return vim.loop.os_uname().sysname == 'Windows'
```

Comment:
This can be a constant value. I think we can safely assume that uname will not change :)

Reply:
Oh right :)